src/main/scala/de/zalando/beard/renderer/BeardTemplateRenderer.scala

```diff
@@ -34,6 +36,14 @@ class BeardTemplateRenderer(templateCompiler: TemplateCompiler) {
     renderResult.write(string)
   }
 
+  private def stringRepresentation(value: Any): String = value match {
+    case null => ""
```

Comment:
of course :)

On Thu, Jan 14, 2016 at 5:57 PM, slavaschmidt <notifications@github.com>
wrote:

> In src/main/scala/de/zalando/beard/renderer/BeardTemplateRenderer.scala
> <https://github.com/zalando/beard/pull/8#discussion_r49752013>:
>
> > @@ -34,6 +36,14 @@ class BeardTemplateRenderer(templateCompiler: TemplateCompiler) {
> >      renderResult.write(string)
> >    }
> >
> > +  private def stringRepresentation(value: Any): String = value match {
> > +    case null => ""
>
> I'd make most of them configurable, not only this one
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zalando/beard/pull/8/files#r49752013>.
>